### PR TITLE
Install devtools from GitHub before other dependencies

### PR DESCRIPTION
**Pull request by juliasilge**
Can we temporarily change r-appveyor to install devtools from GitHub, as discussed in https://github.com/krlmlr/r-appveyor/issues/69?

(FYI, I am not totally sure I have made the right changes to `travis-tool.sh` here, but I made a stab at it.)

### CODE REVIEW

**Comment by juliasilge** [comment]
Hmmmm, well, that did not seem to work, did it?

File: scripts/travis-tool.sh [hunk 1]
```diff
@@ -363,6 +363,8 @@ case $COMMAND in
     ##
     ## Install package dependencies from CRAN (needs devtools)
     "install_deps")
+        InstallGithub "hadley/devtools"
+        ;;
```
Comment (jimhester) [inline scripts/travis-tool.sh]: I believe you need to remove these semicolons, they should only be at the end of each case.

**Comment by krlmlr** [comment]
I like the idea of making everything work out of the box, but I'm afraid this will require Rtools, which isn't currently installed by default.

**Comment by juliasilge** [comment]
Sure, shall I close the PR?

**Comment by krlmlr** [comment]
It would have worked great if https://github.com/hadley/devtools/issues/1282 were implemented. Thanks anyway.

**Pull request closed without merging**